src/lib/device-online-state.ts

```diff
@@ -157,6 +202,23 @@ class DeviceOnlineStateManager {
 				}
 			})
 			.then(() => {
+				setInterval(async () => {
```

Comment:
One problem with this being a `setInterval` is that if for some reason the `rsmq.getQueueAttributesAsync` starts to be slow we could pile them up with no backoff

Reply:
I will implement this 👍